README.md

```diff
@@ -35,12 +35,12 @@ You can jump right into editing this file [here](https://github.com/not-yet-awes
     - [Geospatial Information Systems](#geospatial-information-systems)
     - [Machine Learning](#machine-learning)
     - [Mathematics](#mathematics)
+    - [Computer Vision](#computer-vision)
     - [Native desktop application integrations](#native-desktop-application-integrations)
         - [Microsoft Office](#microsoft-office)
     - [Parsers/Emitters](#parsersemitters)
     - [Personal information management](#personal-information-management)
     - [Rust Toolchain](#rust-toolchain)
-    - [SSH](#ssh)
```

Comment:
SSH category doesn't exist on the page

Reply:
Ah, crap, this is what I get for editing without following my own process. Maybe I need some CI here...thanks!